spark/dl/src/main/scala/com/intel/analytics/bigdl/models/utils/DistriOptimizerPerf.scala

```diff
@@ -105,7 +105,7 @@ object DistriOptimizerPerf {
 
     val sc = new SparkContext(conf)
     Engine.init
-    val broadcast = sc.broadcast(MiniBatch(input, labels))
+    val broadcast = sc.broadcast(MiniBatch[Float](input, labels))
```

Comment:
Intellij used to throw an error to me. But I try to delete Float just now, it works fine.....